workers/unity/Assets/Playground/Scripts/Worker/AndroidClientWorkerConnector.cs

```diff
@@ -0,0 +1,60 @@
+using Improbable.Gdk.Core;
+using Improbable.Gdk.Mobile;
+#if UNITY_ANDROID
+using Improbable.Gdk.Mobile.Android;
+#endif
+using System;
+using UnityEngine;
+
+
+namespace Playground
+{
+    public class AndroidClientWorkerConnector : MobileWorkerConnector, IMobileConnectionController
+    {
+        public string IpAddress { get; set; }
+        public ConnectionScreenController ConnectionScreenController { get; set; }
+
+        [SerializeField] private GameObject level;
+
+        private GameObject levelInstance;
+
+        public async void TryConnect()
+        {
+            await Connect(WorkerUtils.AndroidClient, new ForwardingDispatcher()).ConfigureAwait(false);
+        }
+
+        protected override void HandleWorkerConnectionEstablished()
+        {
+            ConnectionScreenController.OnConnectionSucceeded();
+            WorkerUtils.AddClientSystems(Worker.World);
+
+            if (level == null)
+            {
+                return;
+            }
+
+            levelInstance = Instantiate(level, transform);
+            levelInstance.transform.SetParent(null);
+        }
+
+        protected override void HandleWorkerConnectionFailure()
+        {
+            ConnectionScreenController.OnConnectionFailed();
+        }
+
+        protected override string GetHostIp()
+        {
+#if UNITY_ANDROID
+            if (Application.isMobilePlatform && DeviceInfo.IsAndroidStudioEmulator() && IpAddress.Equals(string.Empty))
```

Comment:
`IpAddress.Equals(string.Empty)` should be `string.Empty.Equals(IpAddress)` as `IpAddress` can be `null`